src/BloomExe/Book/BookStorage.cs

```diff
@@ -866,6 +866,8 @@ private string SanitizeNameForFileSystem(string name)
 				name = name.Replace(c, ' ');
 			}
 			name = name.Trim();
+			if (name.StartsWith("."))
```

Comment:
Dropping the period can result in an empty filename if the book title consists only of a period.  And would need a "while" instead of an "if" to handle pathological titles.